

# assert 断言,在表达式是False的时候,报AssertionError异常
# assert expression1,expression2
# try:
#     assert 1==2,"sucker"
# except AssertionError as e:
#     print(e)

assert 1==2,'sucker'

# if not 1==2:
#     raise AssertionError('sucker')


